### PR TITLE
Jumper T12 board

### PR DESCRIPTION
**Pull request by schumixmd**
Added Jumper T12 as a spearate board.
The definition in many cases is isolated from the Taranis X7 or Taranis Xlite so that it doesn't interfere with original board code. 
The navigation is implemented with Taranis Xlite code without Shift key.
The simulator is in progress.

### CODE REVIEW

File: companion/src/firmwares/boards.cpp [hunk 1]
```diff
@@ -63,6 +63,8 @@ uint32_t Boards::getFourCC(Type board)
     case BOARD_TARANIS_XLITE:
       return 0x3978746F;
     case BOARD_TARANIS_X7:
+      return 0x3678746F;
```
Comment (bsongis) [inline companion/src/firmwares/boards.cpp:66]: no

**Comment by schumixmd** [comment]
pull request will not be merged

**Comment by schwabe** [comment]
@schumixmd I think @bsongis was mainly refering to the fact that each board should have its unique id.

**Comment by schumixmd** [comment]
@schwabe @bsongis @kilrah I may reopen the PR any time, but I see no reason to keep it open if the PR will not be pulled in 2.2. 
If not - please close! If yes - let's discuss what do make sense and what not.

File: companion/src/firmwares/boards.cpp (hunk 1 above)
Comment (schumixmd) [reply companion/src/firmwares/boards.cpp:66]: @bsongis Not sure about the identified format. 0x**3A**78746F fits?

**Pull request closed without merging**